### PR TITLE
removes paramed cargo/aux base/engine/EVA access, adds mining access instead

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

I made this PR because I've recently found the PR that originally added Paramedics https://github.com/tgstation/tgstation/pull/48236 - which says they were only meant to have access to department front doors which is something I think was forgotten about overtime, it's currently very troubling to the point where Paramedics have access to insulated gloves roundstart, while Atomspheric technicians don't, despite not even working in that department.

I also removed their Cargo access, because I don't see any reason why Paramedics should be operating the cargo console, and replaced it with Mining access, which does two things:
- Lets Paramedics enter Cargo without the ability to open the cargo shuttle airlocks
- Makes it somewhat easier for Paramedics to get to the Mining base, since Miners tend to die often.

## Why It's Good For The Game

I often see people play Paramedic solely because;
- if they roll antag, they already have so much access that they don't need to do any work towards stealing any (like cargo smuggle objective with prog. traitors)
- Go Paramedic just to play another job, but with more access. 

Examples of this includes:

- Wanting to make an SM in space, so uses engineering access to get tools/insuls/emitter/ect, then Cargo access to order their own SM crate, rather than just going Engineer and not taking up a Paramedic job slot for someone who actually wanted to recover bodies.
- Abandoning the station to explore space. (Removing EVA hopefully will not cause too much of a problem, since Medbay has a Medical MODsuit they can use whenever needed, and Robotics can build more later if its required.)

Paramedics are one of the most powergamed jobs on the station, and it's very easy to understand why. They have so much access, despite having a grey card, which makes it the easiest way to bypass the trim system, and need for a Silver ID.

Some people tell me that Paramedics should have access to departments so they can fill in for missing roles, but I hate that argument because the Paramedic player should just be playing the department that needs filling in, rather than a Medical job to act like Cargo.

## Changelog

:cl:
balance: Paramedics no longer have access to Auxiliary base, the Supermatter engine room (including insuls), the Cargo shuttle, and EVA. Use a Medical MODsuit instead, you lazy bum!
balance: Paramedics now have access to Mining bay.
/:cl:

### CODE REVIEW

**Comment by GuillaumePrata** [comment]
> Another example could just be Paramedics who abandon the station and explore space, though this PR doesn't really fix that as they still have EVA access, and I don't know how I would fix that without punishing actual Paramedics.

Now that medbay has their own MODsuit the Paramed's EVA access can be sliced off too IMO.

**Comment by JohnFulpWillard** [comment]
> > Another example could just be Paramedics who abandon the station and explore space, though this PR doesn't really fix that as they still have EVA access, and I don't know how I would fix that without punishing actual Paramedics.
> 
> Now that medbay has their own MODsuit the Paramed's EVA access can be sliced off too IMO.

I would, but Medical only spawns with one modsuit, and is usually taken by some doctor since it has 100% bio protection. idk it seems kinda weird to take it away, though comparisons can be made for atmos techs who have to share a single modsuit, but at least atmos techs only share with other atmos techs and not a whole medical department...

**Comment by GuillaumePrata** [comment]
> I would, but Medical only spawns with one modsuit, and is usually taken by some doctor since it has 100% bio protection. idk it seems kinda weird to take it away, though comparisons can be made for atmos techs who have to share a single modsuit, but at least atmos techs only share with other atmos techs and not a whole medical department...

Engineers only have 2 suits on the station while their job is literally to fix the breaches but they don't have EVA access.
If a Paramed needs it for rescue they can ask for the MODsuit or for EVA access, IMO it would mostly cut into the people that join as Paramed to be Space Curator Plus but I guess that is more for the maintainers to decide.

**Comment by ATH1909** [comment]
> > I would, but Medical only spawns with one modsuit, and is usually taken by some doctor since it has 100% bio protection. idk it seems kinda weird to take it away, though comparisons can be made for atmos techs who have to share a single modsuit, but at least atmos techs only share with other atmos techs and not a whole medical department...
> 
> Engineers only have 2 suits on the station while their job is literally to fix the breaches but they don't have EVA access. If a Paramed needs it for rescue they can ask for the MODsuit or for EVA access, IMO it would mostly cut into the people that join as Paramed to be Space Curator Plus but I guess that is more for the maintainers to decide.

the people who join as paramed to be space curator plus would just take that one medbay modsuit and fuck off if you removed their eva access.

**Comment by ATH1909** [comment]
uh, aren't engineers gonna be dying around, well, the _engine_?

**Comment by GuillaumePrata** [comment]
> the people who join as paramed to be space curator plus would just take that one medbay modsuit and fuck off if you removed their eva access.

That is just a empty argument that can be said to any removal in the game.

"If they don't have X by default they will just take X by force."

**Comment by JohnFulpWillard** [comment]
I'm convinced by https://github.com/tgstation/tgstation/pull/64971 and the discussion caused by it saying the Med MODsuit should be standard on all maps, to at least try removing their EVA access as well. I can revert it later if a maintainer has strong opinions on it

**Comment by Serxule** [comment]
you literally need cargo access to easily get to mining though, and also EVA access is good regardless if any bodies are in space.

**Comment by JohnFulpWillard** [comment]
> you literally need cargo access to easily get to mining though
no you don't

> and also EVA access is good regardless if any bodies are in space.
what

**Comment by Serxule** [comment]
> > and also EVA access is good regardless if any bodies are in space.
> 
> what
what is what about this? its pretty common, or even just somewhere depressurized, and its a single med modsuit, that people often take.

**Comment by JohnFulpWillard** [comment]
> what is what about this? its pretty common, or even just somewhere depressurized, and its a single med modsuit, that people often take.

They were given EVA with intent to collect bodies in space, so since they have a medical modsuit, the reason for having it for space-collection is gone, so they don't need one. They can go through the same process as anyone else if they need an EVA space suit for station depressurization, that's unrelated to what is needed for their job.

**Comment by Serxule** [comment]
> > what is what about this? its pretty common, or even just somewhere depressurized, and its a single med modsuit, that people often take.
> 
> They were given EVA with intent to collect bodies in space, so since they have a medical modsuit, the reason for having it for space-collection is gone, so they don't need one. They can go through the same process as anyone else if they need an EVA space suit for station depressurization, that's unrelated to what is needed for their job.

its a single modsuit, unless you can somehow get the CMO's by theft or asking, and since there can be a max of like, 7 meds? not to mention tiders

**Comment by JohnFulpWillard** [comment]
> its a single modsuit, unless you can somehow get the CMO's by theft or asking, and since there can be a max of like, 7 meds? not to mention tiders

https://cdn.discordapp.com/attachments/930713777568088064/943740608642695218/yoCsTXSI.gif

**Comment by Serxule** [comment]
> > its a single modsuit, unless you can somehow get the CMO's by theft or asking, and since there can be a max of like, 7 meds? not to mention tiders
> 
> yoCsTXSI

i will now wait an hour for modsuit modules to be researched

**Comment by jughu** [comment]
They have access to rooms like the tech lathe room because it uses front door access for whatever reason

**Comment by UnokiAs** [comment]
Those are good change, Well I don't see why removing EVA ?
 I had another idea for paramedic access like since their is two slot they would get two different ID(kinda like security), One engineering/service and the other cargo/science would limit the AA feeling while avoiding "Damn I can't get the corpse in the engine room" and instead it would be "Damn I can't get the corpse in the engine room, :m John tider paramedic go get the corpse in engine room"

**Comment by AlinaStarkova** [comment]
> > > what is what about this? its pretty common, or even just somewhere depressurized, and its a single med modsuit, that people often take.
> > 
> > 
> > They were given EVA with intent to collect bodies in space, so since they have a medical modsuit, the reason for having it for space-collection is gone, so they don't need one. They can go through the same process as anyone else if they need an EVA space suit for station depressurization, that's unrelated to what is needed for their job.
> 
> its a single modsuit, unless you can somehow get the CMO's by theft or asking, and since there can be a max of like, 7 meds? not to mention tiders

and engineers and atmos techs when they need suits to do their job and fix breaches but dont have EVA access either...... why should parameds get snowflake access to EVA when the 2 jobs that actually unironically need it dont have it? this is what people mean when they claim parameds keep getting power creep, "other meds will steal the medsuit so give parameds EVA access" and? IC issue, beat up the meds who steal the suit then rather than give them *even. more. access.* over any other job, i wouldnt be surprised if parameds already have more access then the CMO does

**Comment by Lauwma** [comment]
What's funny is that most of the examples given never happen in rounds. 

The only thing that paramedic abuse is the fact they can validhunt by going everywhere to seek the antags.
If you want to nerf that behavior then you must either nerf the tools that are being used for validhunting like people did in the past (baseball bat/syringe gun) or give more reasons for paramedics to do their actual job instead of validhunting

**Comment by UnokiAs** [comment]
> > > > what is what about this? its pretty common, or even just somewhere depressurized, and its a single med modsuit, that people often take.
> > > 
> > > 
> > > They were given EVA with intent to collect bodies in space, so since they have a medical modsuit, the reason for having it for space-collection is gone, so they don't need one. They can go through the same process as anyone else if they need an EVA space suit for station depressurization, that's unrelated to what is needed for their job.
> > 
> > 
> > its a single modsuit, unless you can somehow get the CMO's by theft or asking, and since there can be a max of like, 7 meds? not to mention tiders
> 
> and engineers and atmos techs when they need suits to do their job and fix breaches but dont have EVA access either...... why should parameds get snowflake access to EVA when the 2 jobs that actually unironically need it dont have it? this is what people mean when they claim parameds keep getting power creep, "other meds will steal the medsuit so give parameds EVA access" and? IC issue, beat up the meds who steal the suit then rather than give them _even. more. access._ over any other job, i wouldnt be surprised if parameds already have more access then the CMO does

Removing EVA access feel still kinda meh, Most of the time EVA is open eitheir way or very easy to break into and in the original pr it was INTENDED to paramedic to have access to EVA meanwhile engine access wasn't

**Comment by Lauwma** [comment]
> What's funny is that most of the examples given never happen in rounds. 
> 
> The only thing that paramedic abuse is the fact they can validhunt by going everywhere to seek the antags.
> If you want to nerf that behavior then you must either nerf the tools that are being used for validhunting like people did in the past (baseball bat/syringe gun) or give more reasons for paramedics to do their actual job instead of validhunting 

As always, just another change that tries to adress an issue that doesn't exist with a solution that wouldn't even fix the issue if it existed

**Comment by Lauwma** [comment]
> > > > what is what about this? its pretty common, or even just somewhere depressurized, and its a single med modsuit, that people often take.
> 
> > > 
> 
> > > 
> 
> > > They were given EVA with intent to collect bodies in space, so since they have a medical modsuit, the reason for having it for space-collection is gone, so they don't need one. They can go through the same process as anyone else if they need an EVA space suit for station depressurization, that's unrelated to what is needed for their job.
> 
> > 
> 
> > its a single modsuit, unless you can somehow get the CMO's by theft or asking, and since there can be a max of like, 7 meds? not to mention tiders
> 
> 
> 
> and engineers and atmos techs when they need suits to do their job and fix breaches but dont have EVA access either...... why should parameds get snowflake access to EVA when the 2 jobs that actually unironically need it dont have it? this is what people mean when they claim parameds keep getting power creep, "other meds will steal the medsuit so give parameds EVA access" and? IC issue, beat up the meds who steal the suit then rather than give them *even. more. access.* over any other job, i wouldnt be surprised if parameds already have more access then the CMO does

"Just beat them up" while being based and terry pilled is not a good solution even for LRP.
Not everything should revolve around beating people up especially in the medical department. 
Engineers don't need EVA access because they already have enough suits to cover all the people that needs it (on most Maps), atmo even gets two of them. Paramedics, in comparison, don't get any suits in some maps

**Comment by UnokiAs** [comment]
Does ACCESS_CONSTRUCTION give access to engineering lobby or it was only ACCESS_ENGINE giving them access to the lobby+engine ? (Idk how trim works)

**Comment by JohnFulpWillard** [comment]
> Does ACCESS_CONSTRUCTION give access to engineering lobby or it was only ACCESS_ENGINE giving them access to the lobby+engine ? (Idk how trim works)

Yes it's engineering lobby and techfab. It's what Curators used to have, before Aux base was split into its own access.

**Comment by UnokiAs** [comment]
> > Does ACCESS_CONSTRUCTION give access to engineering lobby or it was only ACCESS_ENGINE giving them access to the lobby+engine ? (Idk how trim works)
> 
> Yes it's engineering lobby and techfab. It's what Curators used to have, before Aux base was split into its own access.

Ah alright thanks

**Comment by Mothblocks** [comment]
I ask the AI to open the EVA shutters literally every time I play MD... as non human. Never had any real problems. I don't see the big deal, honestly.

**Comment by Lauwma** [comment]
> I ask the AI to open the EVA shutters literally every time I play MD... as non human. Never had any real problems. I don't see the big deal, honestly. 

What's the big deal with having the access ?

**Comment by Pickle-Coding** [comment]
> > I ask the AI to open the EVA shutters literally every time I play MD... as non human. Never had any real problems. I don't see the big deal, honestly.
> What's the big deal with having the access ?
Unnecessary access will remove some gameplay elements. Not everything needs to be the easiest and most convenient. Lacking unnecessary access will mean you will have to interact with people or the AI, and that's good. EVA should be managed by heads of staff and AI, not parameds because they need to be in a vacuum.

**Comment by Lauwma** [comment]
> > > I ask the AI to open the EVA shutters literally every time I play MD... as non human. Never had any real problems. I don't see the big deal, honestly.
> >
> > What's the big deal with having the access ?
> 
> Unnecessary access will remove some gameplay elements. Not everything needs to be the easiest and most convenient. Lacking unnecessary access will mean you will have to interact with people or the AI, and that's good. EVA should be managed by heads of staff and AI, not parameds because they need to be in a vacuum.

Heads of staff and should be managing the EVA but aren't doing it. Also that's not true they shouldn't be managing the ability of paramedics to rescue people. Well maybe the CMO would but again, they are not interested in doing it. So you remove the ability to have something, gatekeeping to things people don't wanna do, meaning you just remove the thing.

All servers aren't Manuel, I invite you to have a decent playtime on terry before having insight on how the game should be played.

**Comment by GuillaumePrata** [comment]
> All servers aren't Manuel, I invite you to have a decent playtime on terry before having insight on how the game should be played.

The codebase has been pushing things to be more interactive between crewmembers for so long it is silly that this is still brought up.

I doubt most maintainers would see Terry as the goal on how the game should be played either lol

**Comment by Lauwma** [comment]
> > All servers aren't Manuel, I invite you to have a decent playtime on terry before having insight on how the game should be played.
> 
> 
> 
> The codebase has been pushing things to be more interactive between crewmembers for so long it is silly that this is still brought up.
> 
> 
> 
> I doubt most maintainers would see Terry as the goal on how the game should be played either lol

Forcing interactions and pushing interaction is not the same thing.

What you should be doing is give incentive on interactions, not forcing them by getting rid of things.
It litteraly makes sense that paramedics have access to EVA because in case of an emergency they need to get those suits fast. In hospitals, the emergency responders don't ask the hospital's director if they can send an ambulance, they send it.

**Comment by jughu** [comment]
the point is that they cant walk in by themselves and just pick up everything they need

**Comment by DrDiasyl** [comment]
John fucking Willard, I never thought I will say that.. but you're a hero

**Comment by EricZilla** [comment]
Great change tbh, I've had so many cargo tech rounds ruined by a single paramedic butchering monkies for bounties for 30 minutes straight and constantly getting in the way in cargo, now at least they cant move the fucking shuttle

**Comment by Mothblocks** [comment]
> What's the big deal with having the access ?

Paramedics have an ungodly amount of access and are a popular job among people who don't paramedic. I think they have so much access that having a wallet with paramedic + most heads of staff will prioritize the paramedic, lol. I am definitely interested in relaxing it a ton.

**Comment by UnokiAs** [comment]
> will paramedics be able to use the autolathe?

ACCESS_CONSTRUCTION give access to the engineering lobby+lathe apparently

File: code/datums/id_trim/jobs.dm [hunk 1]
```diff
@@ -361,8 +361,8 @@
 	trim_state = "trim_paramedic"
 	sechud_icon_state = SECHUD_PARAMEDIC
 	extra_access = list(ACCESS_SURGERY)
-	minimal_access = list(ACCESS_AUX_BASE, ACCESS_CARGO, ACCESS_CONSTRUCTION, ACCESS_ENGINE, ACCESS_EVA, ACCESS_HYDROPONICS,
-					ACCESS_MAINT_TUNNELS, ACCESS_MECH_MEDICAL, ACCESS_MEDICAL, ACCESS_MINERAL_STOREROOM, ACCESS_MORGUE, ACCESS_RESEARCH)
+	minimal_access = list(ACCESS_CONSTRUCTION, ACCESS_HYDROPONICS, ACCESS_MAINT_TUNNELS, ACCESS_MECH_MEDICAL,
```
Comment (Mothblocks) [inline code/datums/id_trim/jobs.dm:364]: We really need to make these all multi-lined lists by the way these diffs are annoying to read

**Comment by ZerVexon** [comment]
I agree paramed should have some space suit, I dunno if the fact that all non-CMO personnel getting 1 suit to share is a good idea. Paramedic as intended should be roaming the station, and while they should have a suit, saying "you can have robotics build more" means waiting for research for modules. I'd rather there be a paramed specific MODsuit that features a higher movespeed and less drain for less max complexity, and the roundstart starts with medhud, ion jetpack, and the zero-g grapple. "Search and Rescue" suit, intended for the space or lavaland rescue missions.

**Comment by Pickle-Coding** [comment]
No that's dumb, and PR is already merged.

**Comment by redguy999** [comment]
> * Lets Paramedics enter Cargo without the ability to open the cargo shuttle airlocks
Can't enter cargo, at least on metastation, so that's not right
.

**Pull request merged**